### PR TITLE
test: happy path workflows

### LINKED ISSUE #21: Test happy path workflow

### PR DESCRIPTION
**Pull request by gordsport**
closes #21 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,4 +20,12 @@ if __name__ == '__main__':
 
 ## Set up the action
 
+To configure the GitHub Action...
+
 ## Set up the (beta) project board
+
+To configure the GitHub board...
```
Comment (gordsport-test) [inline README.md:27]: Adding a single review coment

**Review by gordsport-dev — APPROVED** [review]
approving review

File: README.md (hunk 1 above)
Comment (gordsport) [reply README.md:27]: fixed

Comment (gordsport-dev) [inline README.md:27]:
```suggestion
To configure the GitHub board setting in a columns...
```

**Review by gordsport-dev — CHANGES_REQUESTED** [review]

**Review by gordsport-test — APPROVED** [review]

**Review by gordsport-dev — APPROVED** [review]
lgtm

File: .github/workflows/project_board_automation.yml [hunk 2]
```diff
@@ -171,7 +172,7 @@ jobs:
             query($pr_url: URI!) {
               resource(url: $pr_url) {
                 ... on PullRequest {
-                  reviewThreads(first: 100) {
+                  reviewThreads(last: 100) {
```
Comment (gordsport-test) [inline .github/workflows/project_board_automation.yml:175]: is this the correct order?

File: .github/workflows/project_board_automation.yml [hunk 3]
```diff
@@ -193,6 +194,7 @@ jobs:
                 }
               }
             }'  -f pr_url=$PR_URL > data.json
+          echo 'REVIEWS_COUNT='$(jq '[.data.resource.reviewThreads.edges[] | .node | select(.isResolved != null) | length] | add' data.json) >> $GITHUB_ENV
```
Comment (gordsport-test) [inline .github/workflows/project_board_automation.yml:197]: random review comment

**Review by gordsport-test — DISMISSED** [review]

**Review by gordsport-dev — CHANGES_REQUESTED** [review]
test

**Review by gordsport-test — APPROVED** [review]

**Review by gordsport-dev — APPROVED** [review]

**Pull request merged**